zerver/tests/test_bugdown.py

```diff
@@ -217,8 +217,9 @@ class BugdownTest(ZulipTestCase):
     def assertEqual(self, first: Any, second: Any, msg: Text = "") -> None:
         if isinstance(first, Text) and isinstance(second, Text):
             if first != second:
-                raise AssertionError("Actual and expected outputs do not match; showing diff.\n" +
-                                     mdiff.diff_strings(first, second) + msg)
+                diff = mdiff.diff_strings(first, second)
+                raise AssertionError("Actual and expected outputs do not match; showing diff.\n%s%s" %
+                                     (diff if diff is not None else "[Could not perform diff]", msg))
```

Comment:
Hmm, it looks like we only use `mdiff` in tests.

Rather than have it return None in its error case, let's have it just raise an exception. That seems like cleaner behavior anyway -- after all, that's what will happen if the subprocess fails in some unanticipated way -- and there's little risk to worry about in changing it, because it is only in tests.